test/basic.html

```diff
@@ -22,6 +22,17 @@
   <link rel="import" href="../paper-radio-button.html">
 
 </head>
+<style is="custom-style">
+  paper-radio-button.large {
+    --paper-radio-button-size: 48px;
+  }
+  paper-radio-button.medium {
+    --paper-radio-button-size: 24px;
+  }
+  paper-radio-button.small {
+    --paper-radio-button-size: 12px;
+  }
+</style>
```

Comment:
The tests were failing because the height of the radio label container is computed with the radio label in mind, even if the label is not there. Same with the width. Adjusted tests to compensate for that.